src/utils.ts

```diff
@@ -18,3 +18,22 @@ export const proofsToJSON = (
 ): JsonProof[] => {
   return proofs.filter((p) => !!p).map((p) => p?.toJSON()) as JsonProof[];
 };
+
+const isDevEnv = () => {
+  return process.env.NODE_ENV === "development";
```

Comment:
Is this the right way of getting the environment variable? @silimarius

Reply:
Yes, just add a TODO to also make it work with import.meta.ENV in the bundler